tools/local-network/local_network.py

```diff
@@ -461,7 +461,7 @@ def build_binaries(self):
             )
 
         subprocess.run(
-            f'cd {PROJECT_DIR} && CONSENSUS_ENCLAVE_PRIVKEY="{enclave_pem}" cargo build -p mc-consensus-service -p mc-ledger-distribution -p mc-admin-http-gateway -p mc-util-grpc-admin-tool -p mc-mobilecoind -p mc-crypto-x509-test-vectors -p mc-consensus-mint-client -p mc-util-seeded-ed25519-key-gen {CARGO_FLAGS}',
+            f'cd {PROJECT_DIR} && CONSENSUS_ENCLAVE_PRIVKEY="{enclave_pem}" cargo build -p mc-consensus-service -p mc-ledger-distribution -p mc-admin-http-gateway -p mc-util-grpc-admin-tool -p mc-mint-auditor -p mc-mobilecoind -p mc-crypto-x509-test-vectors -p mc-consensus-mint-client -p mc-util-seeded-ed25519-key-gen {CARGO_FLAGS}',
```

Comment:
It seems odd to build this here, when it is not used in this script. Where will this new test be run?

Reply:
It's very useful to have it around for manual testing. I have frequently stood up a local network and used the auditor to try out things. I don't see any reason not to build it, and maybe it some point we'll decide to also run it. I opted to not do that for now since the requirement didn't come up and I didn't want to bloat the already-messy local network script.